packages/eslint-plugin/src/rules/return-await.ts

```diff
@@ -54,6 +56,39 @@ export default util.createRule({
       return false;
     }
 
+    function removeAwait(
+      fixer: TSESLint.RuleFixer,
+      node: TSESTree.ReturnStatement | TSESTree.ArrowFunctionExpression,
+    ): TSESLint.RuleFix | null {
+      const awaitNode =
+        node.type === AST_NODE_TYPES.ReturnStatement
+          ? node.argument
+          : node.body;
+      // Should always be an await node; but let's be safe.
+      if (!awaitNode || awaitNode.type !== AST_NODE_TYPES.AwaitExpression) {
+        return null;
```

Comment:
It looks like this (and the other [`return null`](https://github.com/typescript-eslint/typescript-eslint/pull/1561/files#diff-fcd8de34a7aa234378168246e69fdf56R86)) are what are tanking the patch coverage numbers.

It's not clear to me how I can successfully screw up the AST to cause this to fail—but I am also uncomfortable removing these checks. Curious whether you think I should make a change here?